test-app/server.js

```diff
@@ -144,9 +145,12 @@ const mockUserStore = new MockUserStoreService()
 const requestedReportService = new RequestedReportService(mockUserStore)
 const recentlyViewedService = new RecentlyViewedStoreService(mockUserStore)
 const bookmarkService = new BookmarkService(mockUserStore)
+const downloadPermissionService = new DownloadPermissionService(mockUserStore)
```

Comment:
Will users of our library need to set this up too? If so, please could you add it to the Integration guide?

Reply:
Theres been no talk of download to the sync journey. there currently no message or feedback for in the sync journey. Perhaps its been forgotten about